README.md

```diff
@@ -129,8 +130,8 @@ Agreement](http://goo.gl/forms/QO9UELxM9m).
 
 # Published artifacts
 
-[Docker](https://hub.docker.com/r/camptocamp/mapfish-print)
+[Docker](https://hub.docker.com/r/camptocamp/mapfish_print)
 
-[War and jar from version 3.23](https://github.com/orgs/mapfish/packages)
+[War and jar from version 3.23](https://github.com/mapfish/mapfish-print/releases)
```

Comment:
This change is wrong, the war and jar are in packages, not in releases...

Reply:
The releases also contain the war and jar, besides all other things, like in:

https://github.com/mapfish/mapfish-print/releases/tag/3.28.0

Assets 7
print-cli-3.28.0.zip
103 MB
print-lib-3.28.0-javadoc.jar
2.01 MB
print-lib-3.28.0-sources.jar
527 KB
print-lib-3.28.0.jar
877 KB
print-servlet-3.28.0.war
103 MB
Source code (zip)
Source code (tar.gz) 

Conversely, the "packages" section is really wired:

On https://github.com/mapfish/mapfish-print the link "Packages" on the right side brings you to a general mapfish-page:

Here only three packages of the old 3.26 release are available:

https://github.com/orgs/mapfish/packages?repo_name=mapfish-print

That was the reason I changed this.

What is correct now?

Reply:
> The releases also contain the war and jar, besides all other things, like in:

Oups, I didn't see them because the asses are closed.

> Conversely, the "packages" section is really wired:

Effectively, the GitHub UI can be improved..., but we can fill the description :-)

> Here only three packages of the old 3.26 release are available:

Oups, it seem that the command `./gradlew publish` is dropped by error in this commit ...
https://github.com/mapfish/mapfish-print/commit/b261051bee03220864b61a2b80d98e1f147a0dbf#diff-ca012d969385d0ed162c422aab9ab523ba77567277a4a8bc534dfcf5a6b3c2fcL30

> What is correct now?

The link on packages are required because it's usable with Maven :-)
The release link is optional :-)

Reply:
I added both packages and releases.
You fix the build script for packages?